From: Andreas Metzler <ametzler@debian.org>
Date: Thu, 9 May 2024 13:57:27 +0200
Subject: Do not set AEAD: OCB key preference on new keys.

Origin: vendor
Forwarded: not-needed
Last-Update: 2024-05-09
---
 g10/keygen.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/g10/keygen.c b/g10/keygen.c
index d4fe7aa..fd9f16e 100644
--- a/g10/keygen.c
+++ b/g10/keygen.c
@@ -417,8 +417,12 @@ keygen_set_std_prefs (const char *string,int personal)
 	      strcat(dummy_string,"S7 ");
 	    strcat(dummy_string,"S2 "); /* 3DES */
 
+#if 0
+	    /* Do not set AEAD: OCB on newly generated key, it is a LibrePGP
+	     * feature.  */
             if (!openpgp_aead_test_algo (AEAD_ALGO_OCB))
 	      strcat(dummy_string,"A2 ");
+#endif
 
             if (personal)
               {
